src/components/ComputeModal.js

```diff
@@ -802,7 +805,10 @@ export const ComputeModalBase = ({ onDismiss, onSuccess, runtimes, persistentDis
         ...commonButtonProps,
         onClick: () => {
           applyChanges()
-        }
+        },
+        disabled: isUpdateDisabled,
+        tooltipSide: 'left',
+        tooltip: isUpdateDisabled ? `Cannot perform change on environment in status ( ${currentRuntimeDetails.status} )` : 'Update Environment'
```

Comment:
I'm not sure what the possible values of `currentRuntimeDetails.status` are, but would this read a little more clearly as something like `Cannot update environment in ${currentRuntimeDetails.status} state`?

Reply:
...or `Cannot update environment in ${currentRuntimeDetails.status} status` since `status` is the term we typically use for cloud environments, disks, etc.

Reply:
It's not only updates that are prohibited though based on state, sometimes it's deletion. I'll move the status after though